orderer/common/server/main.go

```diff
@@ -252,7 +251,7 @@ func verifyNoSystemChannelJoinBlock(config *localconfig.TopLevel, cryptoProvider
 		if err != nil {
 			logger.Panicf("Failed unmarshalling join-block for channel '%s', error: %v", channelName, err)
 		}
-		if err = onboarding.ValidateBootstrapBlock(block, cryptoProvider); err == nil {
+		if err = ValidateBootstrapBlock(block, cryptoProvider); err == nil {
```

Comment:
you can change that method to package scope, i.e. ValidateBootstrapBlock -> validateBootstrapBlock